src/icons/arrow-right-active.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="100%" width="100%" fill="#173552" viewBox="0 0 24 24"><title>keyboard-arrow-right</title><path d="M23.707,11.291,18.929,6.513a1,1,0,0,0-1.707.707v3.028a.25.25,0,0,1-.25.25H1.5a1.5,1.5,0,0,0,0,3H16.972a.25.25,0,0,1,.25.25v3.028a1,1,0,0,0,1.707.707l4.778-4.778a1,1,0,0,0,0-1.414Z"/></svg>
```

Comment:
It's easier to add icons just once, and then change the attributes with css. That way, we don't have to maintain multiple instances of the same icon, and if, for example, the color scheme changes, we don't have to go through all the icons to adjust the colors by hand.

Reply:
Done for the arrow icon.

For the other icons:
How do I set the title attribute through CSS, in the context of accessibility? How do I set a specific fill from a multi-coloured icon?

Reply:
Setting the title is not possible through CSS as far as I know. 

You can add classes to the SVG (e.g. `border` and `background`) to target a specific element within the image.